### DISCUSSION TITLE
SimpleRateThrottle Docstring

### DISCUSSION THREAD

**Discussion by juanbeniteza** [encode/django-rest-framework#8143] [General]
I'm trying to implement some throttles for my API, I will like to use the `SimpleRateThrottle`  class but with different rates per view. I read in the docs this 
![image](https://user-images.githubusercontent.com/22871775/131063509-8bd2b7be-9df8-4e6b-adda-15bce9d351b3.png)

 I suppose this means that we can set the `rate` attribute on the View and we can customize the rates per view, something like 
![image](https://user-images.githubusercontent.com/22871775/131063738-c3224a7c-e7c1-42c0-8556-ed9b6d8b74eb.png)

Of course, this doesn't work, I'm getting an `ImproperlyConfigured` exception and this is because the implementation expects the `rate` to be an attribute of the `SimpleRateThrottle` class.

My question here is, shouldn't the docs say this instead of saying that `rate` is an attribute on View class?

**Comment by lovelydinosaur** [comment]
Yes, having had a look myself that seems like a good catch.
We'd happily accept a pull request changing the wording there.
Should be "on the Throttle class" or something similar, right?

**Reply by juanbeniteza** [reply]
Yes, I think "on the Throttle class" would be ok.

I will create the pull request.
